proto/v1/stream.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package protov1;
+
+import "opencdc/v1/opencdc.proto";
+
+service StreamService {
```

Comment:
How about `SourceService` and `DestinationService`?

Reply:
but `SourceService` would still be used by a source and a destination, which is still a bit confusing

Reply:
I proposed it because I see the service as an interface to the server connector (it's the one that implements the service). So a source server connector exposes a `SourceService`, to which a destination client connector can connect.

Reply:
oh I see, yeah this makes sense, so the one already implemented would be the `SourceService`.
what about the RPC method, should we keep it as `stream`?

Reply:
I think `Stream` sounds good 👍

Reply:
rename is pushed, and created one for the client https://github.com/conduitio-labs/conduit-connector-grpc-client/pull/6